pkg/fqdn/dnsproxy/proxy_test.go

```diff
@@ -1028,39 +1029,79 @@ func (t selectorMock) String() string {
 }
 
 func Benchmark_perEPAllow_setPortRulesForID(b *testing.B) {
+	// This benchmark loops through nEPs number of endpoints ~to distinct pods;
+	// - With nMatchPatterns number of matchPatterns in their DNS config
+	// - With nMatchName number of matchNames in their DNS rules
+	// - Where every everyNIsEqual EPs have the same DNS rules
+	// - Where every everyNHasWildcard EPs have a wildcard inserted
 	const (
-		nMatchPatterns = 100
+		nEPs              = 10000
+		nEPsAtOnce        = 60
+		nMatchPatterns    = 30
+		nMatchNames       = 600
+		everyNIsEqual     = 10
+		everyNHasWildcard = 20
+		cacheSize         = 128
 	)
+	re.InitRegexCompileLRU(cacheSize)
+	runtime.GC()
+	runtime.GC()
```

Comment:
Why twice?

Reply:
Got used to running it multiple times since I saw that some times one didn't actually gc all the things, making the measurements a bit unstable. I'll sanity check it and remove it in case one is enough!

Reply:
Yeah, got the same with one, so removed it. Thanks!